controllers/clustergroupupgrade_controller.go

```diff
@@ -1770,8 +1871,23 @@ func (r *ClusterGroupUpgradeReconciler) SetupWithManager(mgr ctrl.Manager) error
 	})
 
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&ranv1alpha1.ClusterGroupUpgrade{}).
-		Owns(placementRuleUnstructured).
+		For(&ranv1alpha1.ClusterGroupUpgrade{}).WithEventFilter(predicate.Funcs{
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			// This is a hack so we can ignore CGU status update only
```

Comment:
Maybe we need a Jira to track this issue and remove the hack one the issue is solved?

Reply:
The issue was raised 5 years ago and still no fix planned. Realistically, it probably will never get fixed. So I am not sure if adding a jira will help us.